arangod/Cluster/ClusterFeature.cpp

```diff
@@ -662,7 +662,10 @@ void ClusterFeature::stop() {
     // log off the server from the agency, without permanently removing it from
     // the cluster setup.
     ServerState::instance()->logoff(10.0);
-  } 
+  }
+
+  // make sure ClusterInfo's syncer threads have stopped.
```

Comment:
Does it make sense to additionally notify the two SyncerThreads here, in case they missed the `beginShutdown()` somehow? This is theoretically possible if for whatever reason `beginShutdown()` is not executed for them.